meson.build

```diff
@@ -27,10 +27,13 @@ if api_bus == 'user'
    common_cflags += '-DUSE_USER_API_BUS'
 endif
 
-# Set option of inih libraries maximum line length
+# inih library options
+# Set option of maximum line length
 # 500 characters for actual value
 # 20 characters as buffer (key, library internal usage, etc.)
 common_cflags += '-DINI_MAX_LINE=520'
```

Comment:
Is it possible to increase to 1000 or 1500 instead of 500 ?

Reply:
Sure, 500 is just an arbitrary value like 1000 or 1500. It should be bigger than the max dns name length, though. Its just a memory size will definitely been allocated, so I'd not have it unreasonably large considering the multiline support.

Reply:
right